### ISSUE TITLE
Reload ban file on join

### ISSUE THREAD

**Issue by Tsin** [teeworlds/teeworlds#1096]
everytime someone tries to join in, server should reload or check the ban file

it will help in managing multi servers that use one ban file for all servers

**Comment by minus7** [comment]
I don't think this is a good idea. If you want shared bans, something should push them into the server, e.g. via econ.

**Comment by heinrich5991** [comment]
Or some mutual bans server.

**Comment by minus7** [comment]
If you're suggesting a server that is queried to check if an IP is banned, #1136 may come into play.
